factories/resolverFactory.js

```diff
@@ -56,6 +56,8 @@
           },
           size: ids.length,
         };
+      } else if ( settings.searchEngine === 'vectara') {
+        // Vectara does not have an endpoint to retrieve per doc metadata directly
```

Comment:
Nice comment!

Reply:
BTW, I was struggling with that as well, that we make a LOT more assumptions of what search engines will do ;-)....

Reply:
so, I could maybe test this via firing up Quepid and giving it a work out...  and try and make some edits and see if anything break.s..